bindings/wasm/lib/level-of-detail.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright 2025 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {getManifoldModuleSync} from './wasm.ts';
+
+/**
+ * Control detail level for the
+ * {@link manifold-encapsulated-types!CrossSection.circle CrossSection.circle},
+ * {@link manifold-encapsulated-types!Manifold.cylinder Manifold.cylinder},
+ * {@link manifold-encapsulated-types!Manifold.sphere Manifold.sphere}, and
+ * {@link manifold-encapsulated-types!Manifold.revolve Manifold.revolve}
+ * constructors.
+ *
+ * Libraries should not change these values, and if run through manifoldCAD or
+ * the manifoldCAD CLI, will not be able to.  Libraries may get values to
+ * determine their own level of detail.
+ *
+ * @packageDocumentation
+ */
+
+let minCircularAngle: number = 10.0;
+let minCircularEdgeLength: number = 1.0;
```

Comment:
Why store these again here? Is it just to enable e.g. `getMinCircularAngle`? Can you give me a use case for that getter?